Marlin/src/module/motion.cpp

```diff
@@ -1290,7 +1290,7 @@ feedRate_t get_homing_bump_feedrate(const AxisEnum axis) {
 /**
  * Home an individual linear axis
  */
-void do_homing_move(const AxisEnum axis, const float distance, const feedRate_t fr_mm_s=0.0) {
+void do_homing_move(const AxisEnum axis, const float distance, const feedRate_t fr_mm_s=0.0, const bool heaters_off=true) {
```

Comment:
It seems a little strange to have an argument called `heaters_off` which _may or may not_ impact the execution of the function.

If this were named something like `final_move` then it would not misrepresent what is happening, even if the value is ignored.

Reply:
I was trained to always leave a tidbit in a submission. By all means, I heartily agree to change that to "final_approach` for example.
Wico ASAP